libV2/tasks/ManualAdjust.js

```diff
@@ -0,0 +1,89 @@
+const async = require('async');
+const BaseTask = require('./BaseTask');
+const UtapiClient = require('../client');
+const { LoggerContext } = require('../utils');
+
+const logger = new LoggerContext({
+    module: 'ManualAdjust',
+});
+
+function collectArgs(arg, prev) {
+    return prev.concat([arg]);
+}
+
+class ManualAdjust extends BaseTask {
+    async _setup() {
+        // Don't include default flags
+        await super._setup(false);
+        this._program
+            .option('-h, --host <host>', 'Utapi server host', 'localhost')
+            .option('-p, --port <port>', 'Utapi server port', '8100', parseInt)
+            .option('-b, --bucket <buckets...>', 'target these buckets', collectArgs, [])
+            .option('-a, --account <accounts...>', 'target these accounts', collectArgs, [])
+            .option('-u, --user <users...>', 'target these users', collectArgs, [])
+            .requiredOption('-o, --objects <adjustment>', 'adjust numberOfObjects by this amount', parseInt)
+            .requiredOption('-s, --storage <adjustment>', 'adjust storageUtilized by this amount', parseInt);
+    }
+
+    async _start() {
+        this._utapiClient = new UtapiClient({
+            host: this._program.host,
+            port: this._program.port,
+            disableRetryCache: true,
+        });
+        await super._start();
+    }
+
+    async _pushAdjustmentMetric(metric) {
+        logger.info('pushing adjustment metric', { metric });
+        await this._utapiClient.pushMetric(metric);
+    }
+
+    async _execute() {
+        const timestamp = Date.now();
+
+        const objectDelta = this._program.objects;
+        const sizeDelta = this._program.storage;
+
+        if (!objectDelta && !sizeDelta) {
+            throw Error('You must provided at least one of --objects or --storage');
+        }
```

Comment:
correct me if I am wrong, `--storage` and `--objects` are required options, so this check may be redundant.

Reply:
I must have discovered `requiredOption` after this